jobs/create-initial-subgroup-branch.js

```diff
@@ -25,7 +25,10 @@ module.exports = async function ({ repositoryId, groupName }) {
   const log = Log({logsDb: logs, accountId, repoSlug: repoDoc.fullName, context: 'create-initial-subgroup-branch'})
 
   log.info('started')
-  if (repoDoc.id === '135286129') return
+  if (repoDoc.packages && Object.keys(repoDoc.packages).length > 300) {
```

Comment:
this looks like a good candidate to be moved into a utility function.

something like `if(!notTooManyPackageJSONFiles(repoDoc)) { return } ` obvs, with a better name :)